src/Resource/Listing.php

```diff
@@ -300,4 +300,266 @@ public function deletePaymentMethod($listingId, $paymentMethodId)
+
+    /**
+     * @param int                          $listingId
+     * @param \Traum\Entity\ListingArrival $listingArrival
+     * @return \Traum\Entity\ListingArrival
+     * ToDo: post collection or entity? (https://github.com/theDisco/traum/issues/7)
+     */
+    public function postArrivals($listingId, Entity\ListingArrival $listingArrival)
+    {
+        $uri = sprintf('/listing/%d/arrival', $listingId);
+        $body = $this->executePost($uri, $listingArrival, new Transformer\ListingArrival);
+
+        return new Entity\ListingArrival($body);
+    }
+
+    /**
+     * @param int $listingId
+     * @return bool
+     * @throws \Traum\Exception\InvalidRequest
+     */
+    public function deleteArrivals($listingId)
+    {
+        $uri = sprintf('/listing/%d/arrival', $listingId);
+        $response = $this->request('DELETE', $uri);
+
+        return $response->getStatusCode() === 204;
+    }
+
+    /**
+     * @param int $listingId
+     * @param int $arrivalId
+     * @return \Traum\Entity\ListingArrival
+     */
+    public function getArrival($listingId, $arrivalId)
+    {
+        $uri = sprintf('/listing/%d/arrival/%d', $listingId, $arrivalId);
+        $body = $this->executeGet($uri);
+
+        return new Entity\ListingArrival($body);
+    }
+
+    /**
+     * @param int $listingId
+     * @param int $arrivalId
+     * @return \Traum\Entity\ListingArrival
+     */
+    public function deleteArrival($listingId, $arrivalId)
+    {
+        $uri = sprintf('/listing/%d/arrival/%d', $listingId, $arrivalId);
+        $response = $this->request('DELETE', $uri);
+
+        return $response->getStatusCode() === 204;
+    }
+
+    /**
+     * @param int $listingId
+     * @return \Traum\Entity\ListingSuitabilityCollection
+     */
+    public function getSuitabilities($listingId)
+    {
+        $uri = sprintf('/listing/%d/suitability', $listingId);
+        $body = $this->executeGet($uri);
+
+        return new Entity\ListingSuitabilityCollection($body);
+    }
+
+    /**
+     * @param int                              $listingId
+     * @param \Traum\Entity\ListingSuitability $listingSuitability
+     * @return \Traum\Entity\ListingSuitability
+     * ToDo: post collection or entity? (https://github.com/theDisco/traum/issues/7)
+     */
+    public function postSuitabilities($listingId, Entity\ListingSuitability $listingSuitability)
+    {
+        $uri = sprintf('/listing/%d/suitability', $listingId);
+        $body = $this->executePost($uri, $listingSuitability, new Transformer\ListingSuitability);
+
+        return new Entity\ListingSuitability($body);
+    }
+
+    /**
+     * @param int                              $listingId
+     * @param \Traum\Entity\ListingSuitability $listingSuitability
+     * @return \Traum\Entity\ListingSuitability
+     * ToDo: post collection or entity? (https://github.com/theDisco/traum/issues/7)
+     */
+    public function patchSuitabilities($listingId, Entity\ListingSuitability $listingSuitability)
```

Comment:
To be honest I have no idea how it is supposed to work. Based one example in the documentation the only difference between this and the entity is the identifier. @CreativeNative maybe you can ask the devs to shed some light and enlighten us?